.gitignore

```diff
@@ -1,3 +1,75 @@
+## Linux ##
```

Comment:
i'm starting to think there should be global gitignores people are advised to use (its a pain to have all the repos ignore all the folders of all the tools)

Reply:
Agreed.

I don't really see how this has anything to do with this repo personally; users should be more then capable of managing their own 😂

Reply:
Honestly I've seen many repos including them (even urllib3 if I recall correctly) so I thought I should've add it